index.md

```diff
@@ -5,4 +5,4 @@ nav_order: 1
 has_children: true
 ---
 
-Harry Edward Styles – brytyjski bard urodzony 1 lutego 1994 w Redditch, pieśniarz, autor tekstów, muzyk, model i aktor. Karierę muzyczną rozpoczął w 2010 od udziału w siódmej edycji brytyjskiego programu typu talent show The X Factor. W latach 2010-2015 jeden z wokalistw boys bandu One Direction co znaczy "jeden kierunek"(xD bez sensu)
\ No newline at end of file
+Harry Edward Styles – brytyjski bard urodzony 1 lutego 1994 w Redditch, pieśniarz, autor tekstów, muzyk, model i aktor. Karierę muzyczną rozpoczął w 2010 od udziału w siódmej edycji brytyjskiego programu typu talent show The X Factor. W latach 2010-2015 jeden z wokalistw boys bandu One Direction co znaczy "jeden kierunek"(xD bez sensu). Harry Styles jest tez komikiem i szachistą.
```

Comment:
Nie no co ty stara? Jaki komik? Jaki szachista? Toc to Wałesa nie jest?

Reply:
No tak, nie wiedziałeś? Lessons to be learned. Doucz się pozdro.

Reply:
Okaaaay